static/js/components/PersonalTab.js

```diff
@@ -23,15 +39,74 @@ class PersonalTab extends React.Component {
     ui:             UIState,
     nextStep:       () => void,
     prevStep:       () => void,
+    dispatch:       Dispatch,
+    dashboard:      DashboardState,
+    enrollments: ProgramEnrollmentsState,
+    currentProgramEnrollment:    ProgramEnrollment,
   };
 
+  programListing = (programs: Program) => {
+    const makeMenuItems = R.map(program => (
+      <MenuItem value={program.id} key={program.id} primaryText={program.title} />
+    ));
+    const sortPrograms = R.sortBy(R.compose(R.toLower, R.prop('title')));
+    const menuItems = R.compose(makeMenuItems, sortPrograms);
+    return menuItems(programs || {}) ;
+  }
+
+  setProgramHelper = (event: Event, key: string, value: string) => {
+    const {
+      dispatch,
+      enrollments: { programEnrollments },
+    } = this.props;
+    let selected = programEnrollments.find(enrollment => enrollment.id === value);
+    dispatch(setCurrentProgramEnrollment(selected));
+  }
+
+  selectProgram = (setProgram: Function, program: Program) => {
+    const { dashboard: { programs } } = this.props;
+
+    return (
+      <SelectField
+        style={{width: "65%"}}
+        hintText={program ? program.title : "Select Program"}
+        onChange={setProgram}
+        value={program.id}
```

Comment:
When I create a new user I am getting a failure. It looks like it is caused by `program` being falsey on this line